source/mongodump.txt

```diff
@@ -224,6 +224,18 @@ Syntax
    .. tab:: Connect to a Sharded Cluster
       :tabid: connect-to-a-sharded-cluster
 
+      .. note::
+
+         To avoid data inconsistencies, pause the following actions on your
+         sharded cluster when you run ``mongodump``:
+         
+         - Cross-shard transactions
+         - Data definition language operations (operations that create and modify
+           collections)
+         - Chunk balancing 
```

Comment:
Could we instead advise users to pause all writes and turn off balancers? Certain operations may use transactions implicitly under the hood. There could also be race conditions between turning off writes and finishing any in-flight operations, so users should make certain that the cluster has fully quiesced before attempting any snapshots. Ditto this comment for mongoexport as well

Reply:
Hey @ltran-mdb2 just wanted to follow-up on this. Let me know if this needs a separate ticket - thanks!

Reply:
Hi @mvankeulen94 , sorry for the delay! Could you create a separate ticket for this?

Reply:
Filed https://jira.mongodb.org/browse/DOCSP-41895, thank you! Please feel free to reassign if needed

Reply:
Ideally rather than having this blurb here, we could instead restate the disclaimer that these tools are not intended for taking backups, and to see https://www.mongodb.com/docs/manual/administration/backup-sharded-clusters/ for how to back up sharded clusters.